.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
+---
+<!--
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete fields you cannot populate and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+Mandatory fields are marked with the word MANDATORY for your convenience.
```

Comment:
No, "mandatory" fields are absolutely not "for your convenience", but because we ultimately want some boilerplate to be filled in with meaningful text. There's nothing "convenient" about that, and, again, we really should tone down this demanding rhetorics and we more welcome.

It's supposed to be a collaborative project that is, first of all, fun. Biting the bullet, going through forms and jumping through the bureaucratic hoops is not.

Reply:
They are marked with big MANDATORY word for user's convenience at least in the first block: in the case of the boilerplate block it can be ommitted at all, sending the user to style guide and docs. But almost noone reads them.

>It's supposed to be a collaborative project that is, first of all, fun.

Definitely.